sdk-lib/src/main/rust/lib.rs

```diff
@@ -947,21 +952,16 @@ pub unsafe extern "C" fn Java_cash_z_ecc_android_sdk_tool_DerivationTool_deriveT
     let res = panic::catch_unwind(|| {
         let network = parse_network(network_id as u32)?;
         let tsk_wif = utils::java_string_to_rust(&env, secret_key);
-        let sk:SecretKey = (&Wif(tsk_wif)).try_into().expect("invalid private key WIF");
-        let taddr =
-            derive_transparent_address_from_secret_key(&sk)
-                .encode(&network);
+        let sk: SecretKey = (&Wif(tsk_wif)).try_into().expect("invalid private key WIF");
```

Comment:
just commenting:
WIF will be removed on the latest rust 👍  code we will pull in later.

Reply:
Yep; this is why I wanted `rustfmt` run first, so this kind of change would be clearer.